scripts/common.py

```diff
@@ -53,7 +53,11 @@
 
 # Versions of libraries used in backend.
 PILLOW_VERSION = '6.2.2'
-REDIS_CLI_VERSION = '6.0.6'
+# Redis 6.0.5 not latest stable build of redis (6.0.6) because there is a
+# 'make test' bug where the solution has not been released explained in this
```

Comment:
... been released. This is explained ...

Reply:
Done.